src/main/resources/assets/endermail/lang/ru_ru.json

```diff
@@ -0,0 +1,34 @@
+{
+    "_comment": "Ender Mail Language File",
+    "_comment": "Russian - Russian Federation",
+
+    "_comment": "БЛОКИ",
+    "block.endermail.package": "Посылка",
+    "block.endermail.locker": "Ячейка",
+
+    "_comment": "ПРЕДМЕТЫ",
+    "item.endermail.package_controller": "Контроллер посылки",
+    "item.endermail.packing_tape": "Скотч",
+    "item.endermail.stamp": "Марка",
+
+    "_comment": "СТРОКИ",
+    "string.endermail.cancel": "Отмена",
+    "string.endermail.close": "Закрыть",
+    "string.endermail.confirm": "Утвердить",
+    "string.endermail.coordinates": "Координаты",
+    "string.endermail.deliver": "Доставить",
+    "string.endermail.delivered": "Доставлено",
+    "string.endermail.deliveredToLocker": "Отправлено в ячейке",
+    "string.endermail.delivering": "Доставление",
+    "string.endermail.deliveryDistance": "Расстояние доставки",
+    "string.endermail.deliveryLocation": "Место доставки",
+    "string.endermail.error": "Произошла ошибка, переведено сообществом модовестник",
```

Comment:
"An error occured, *translated by modovestnik community*"
Advertisement of his VK community, just like in any other of his "translations"

Reply:
Thanks for the heads up, just removed this.